### DISCUSSION TITLE
[4.13]openshift-install, Additional VMware vSphere configuration parameters : platform.vsphere.apiVIPs

### DISCUSSION THREAD

**Discussion by Hyun-June-Choi** [okd-project/okd#1661] [Q&A]
https://docs.okd.io/4.13/installing/installing_vsphere/installing-restricted-networks-vsphere.html#installation-configuration-parameters-additional-vsphere_installing-restricted-networks-vsphere

[itdev@bastion ~]$ ./openshift-install create install-config --dir=./*****-dev
? SSH Public Key /home/itdev/.ssh/id_ed25519_*****_dev.pub
? Platform vsphere
? vCenter *.*.*.***
? Username admin1@vsphere.local
? Password [? for help] **********
INFO Connecting to vCenter *.*.*.***              
INFO Defaulting to only available datacenter: NTT 
? Cluster /NTT/host/dev
? Default Datastore /NTT/datastore/.../de4000_datastore_1
INFO Defaulting to only available network: vlan290 
X Sorry, your reply was invalid: "10.25.90.41,10.25.90.42,10.25.90.43" is not a valid IP
? The VIP to be used for the OpenShift API.
? Virtual IP Address for API 

in manual
platform.vsphere.apiVIPs 
Virtual IP (VIP) addresses that you configured for control plane API access. 
Multiple IP addresses

I want to know how to enter multiple Virtual IP (VIP) addresses

**Comment by jcpowermac** [comment]
The TUI doesn't take mulitple entries on the vips. Create the install-config like you are doing, only add a single ipv4 address, modify the install-config.yaml after `create install-config` is complete.

**Reply by Hyun-June-Choi** [reply]
ip list format ,(comma) seperate right?
ex > 10.25.*.41,10.25.*.42,10.25.*.43

**Reply by jcpowermac** [reply]
In the terminal ui (TUI) it can't take a string with multiple ip addresses separated by a comma.
see: https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/vsphere/vsphere.go#L362-L397

Create the install-config like you are doing, only use a single ipv4 address, modify the install-config.yaml after create install-config is complete.

**Reply by jcpowermac** [reply]
I am also unsure that you can use multiple ipv4 addresses 
based on the docs I _think_ you can only use one ipv4 and one ipv6

https://docs.openshift.com/container-platform/4.13/installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.html#modifying-install-config-for-dual-stack-network_installing-vsphere-installer-provisioned-network-customizations
